microsoft-graph.d.ts

```diff
@@ -1,10 +1,11 @@
-// Project: https://github.com/microsoftgraph/msgraph-beta-typescript-typings
+// Project: https://github.com/microsoftgraph/msgraph-typescript-typings
```

Comment:
Why this change?

Reply:
@nikithauc I think this will need to be fixed in the generator. Do you have an issue for this?

Reply:
https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/issues/590 Will fix here. Closing this PR because the of the bug the validation fails